services/client/util.go

```diff
@@ -194,3 +211,22 @@ func (su shardByIDAscending) Less(i, j int) bool {
 func (su shardByIDAscending) Swap(i, j int) {
 	su[i], su[j] = su[j], su[i]
 }
+
+func adKey(sid services.ServiceID, id string) string {
+	return fmt.Sprintf("[%s][%s]", serviceKey(sid), id)
```

Comment:
Can we make these format consts? e.g.,
````
const (
    advertisementKeyFmt = "[%s][%s]"
    placementKeyFmt = "%s/%s"
    metadataKeyFmt = "%s/%s"
    serviceKeyFmt = "[%s]%s"
) 
```

Also out of curiosity, is there a reason why we are using different formats for different types of keys?